toolset/benchmark/fortune_html_parser.py

```diff
@@ -74,20 +74,30 @@ def handle_charref(self, name):
     # Not sure why some are escaping '/'
     if val == "47" or val == "047" or val == "x2f":
       self.body.append("/")
+    # "&#40;" is a valid escaping of "(", but
+    # it is not required, so we need to normalize for out
+    # final parse and equality check.
+    if val == "40" or val == "040" or val == "x28":
```

Comment:
I'm seeing some Fortune fails on some tests for Fortune 11 (the Javascript alert that should be escaped). Could you try locally removing the additions to this file and running a test to see if these lines are what is affecting frameworks like [C++/ffead-cpp](https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/151374139#L2674) or [C++/treefrog-mongodb](https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/151374134#L3081)? It's possible that those frameworks are the problem and need to implement more rigorous sanitation on the DB values, but I'd like to figure out if the two things are somehow connected before merging this in.